### PR TITLE
feat: start using new simint api

### PR DESCRIPTION
**Pull request by abdullah-cognite**

### CODE REVIEW

File: Cognite.Simulator.Tests/UtilsTests/SimulationRunnerTest.cs [hunk 1]
```diff
@@ -451,15 +436,15 @@ public async Task TestSimulationRunnerBaseWithApi()
...
+                Assert.True(simRunUpdated.EventId.HasValue);
+                var simEvent = await cdf.Events.GetAsync(simRunUpdated.EventId.Value, source.Token);
+                Assert.NotNull(simEvent);
 
                 Assert.Equal(configObj.CalculationName, simRunUpdated.RoutineName);
                 Assert.Equal("PROSPER", simRunUpdated.SimulatorName);
                 Assert.Equal("Connector Test Model", simRunUpdated.ModelName);
-                Assert.StartsWith("No data points were found for time series", simRunUpdated.StatusMessage);
+                Assert.StartsWith("Calculation ran to completion", simRunUpdated.StatusMessage);
+                Assert.Equal(SimulationRunStatus.success, simRunUpdated.Status);
```
[the remark this answers is not in the document]
Comment (abdullah-cognite) [reply Cognite.Simulator.Tests/UtilsTests/SimulationRunnerTest.cs]: I have removed  the`TestSimulationRunnerBaseWithApi` test and just kept `TestSimulationRunnerBase`

File: Cognite.Simulator.Utils/SimulationRunnerBase.cs [hunk 2]
```diff
@@ -205,9 +178,18 @@ public async Task Run(CancellationToken token)
...
 
+                // sort by event time
+                allEvents.Sort((e1, e2) => {
+                    return e1.Run.CreatedTime > e2.Run.CreatedTime ? -1 : 1;
+                });
+
+                // // get the first 3 events
+                // allEvents = allEvents.Take(3).ToList();
+                
+
```
Comment (polomani) [inline Cognite.Simulator.Utils/SimulationRunnerBase.cs:192]:
```suggestion
```

File: Cognite.Simulator.Tests/UtilsTests/SimulationSchedulerTest.cs [hunk 3]
```diff
@@ -66,25 +94,31 @@ public async Task TestSimulationSchedulerBase()
...
+                            Status = SimulationRunStatus.ready
+                        }
+                    }, source.Token).ConfigureAwait(false);
+
+                Assert.NotEmpty(events.Items);
+                // order events.Items by created time in descending order and filter only items created in the last 10 seconds
+                var latestEvents = events.Items.OrderByDescending(e => e.CreatedTime).ToList();
+                var latestEventsFiltered = latestEvents.Where(e => e.CreatedTime > DateTimeOffset.UtcNow.AddSeconds(-10).ToUnixTimeMilliseconds()).ToList();
+                Assert.Contains(latestEventsFiltered, e => e.RunType == SimulationRunType.scheduled);
+                //eventIds.AddRange(events.Select(e => e.ExternalId));
```
Comment (polomani) [inline Cognite.Simulator.Tests/UtilsTests/SimulationSchedulerTest.cs:112]:
```suggestion
```

**Review by polomani — APPROVED** [review]
Left some minor comments, LGTM

**Pull request merged**